Corona-Warn-App/src/main/java/de/rki/coronawarnapp/bugreporting/debuglog/LogLine.kt

```diff
@@ -13,15 +15,33 @@ data class LogLine(
 
     fun format(): String {
         val time = Instant.ofEpochMilli(timestamp)
-        return "$time  ${priorityLabel(priority)}/$tag: $message\n"
+
+        val baseLine = "$time ${priorityLabel(priority)}/$tag: $message\n"
+
+        return if (throwable != null) {
+            baseLine + "\n" + getStackTraceString(throwable)
+        } else {
+            baseLine
+        }
```

Comment:
We previously overlooked that the actual stacktrace was not written to log :frowning: